html-forms/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>html-forms</title>
+  </head>
+  <body>
+    <section>
+      <h1><strong>Coding Form!</strong></h1>
+      <p id="description">description</p>
+      <p></p>
+      <form id="survey-form">
+        <label id="email-label">Email<input id="email" /></label>
+        <label>why is coding cool?</label>
+        <select id="why-is-coding-cool?">
+          <option>Select an option</option>
+          <option>Because it's intresting</option>
+          <option>Because it's fun</option>
+          <option>Because it's powerful</option>
+        </select>
+        <p>
+          <label>Favortie Language (choose one)</label>
```

Comment:
Since this `label` isn't associated with an `input`, the text should just be directly inside the `p`